wagtail/wagtailadmin/views/pages.py

```diff
@@ -504,6 +504,25 @@ def unpublish(request, page_id):
     })
 
 
+def unshare(request, page_id):
+    page = get_object_or_404(Page, id=page_id).specific
+    if not page.permissions_for_user(request.user).can_unshare():
+        raise PermissionDenied
+
+    if request.method == 'POST':
+        page.unpublish()
+
```

Comment:
just trying to understand the scenarios:

#-- if page is shared & live
- then it is removed from the content site but is still visible on the www site.
- they will need to do an extra step of unpubishing it on www 

if page is unshared & not live
- then they can unshare it which removes it from being visible on the content site

is this correct? @kurtw